src/main.rs

```diff
@@ -139,6 +139,35 @@ enum AutofixMode {
     Yolo,
 }
 
+fn prelude(suggestion: &Replacement) {
+    let snippet = &suggestion.snippet;
+    if snippet.text.1.is_empty() {
+        if suggestion.replacement.ends_with('\n') || suggestion.replacement.starts_with('\n') {
+            println!("{}", "Insert line:".yellow().bold());
+        } else {
+            println!("{}", "At:".yellow().bold());
+            println!(
+                "{lead}{text}{tail}",
+                lead = indent(4, &snippet.text.0),
```

Comment:
Memo for later refactoring: Turn `text` from `(String, String, String)` into `struct ReplacementText { leading: String, content: String, trailing: String, }`